src/common/constants/Dependency.ts

```diff
@@ -1,5 +1,4 @@
 export enum DEPENDENCY {
-  MIGRATION = 'migrationService',
   LOCATOR = 'DependencyLocator',
```

Comment:
There's now only one usage of `DEPENDENCY.FORMAT_SERVICE` if the intention is to deprecate the enum, now might be a time to do it. I see that there's a lot of entries in the enum though, so unsure of the intention

Reply:
I have been struggling a little to find the best way, but I think by using a discriminated union of the dependency keys, it is easier to type the return of the dep locator and also get autocomplete of the string, so yeah moving toward deprecating it